packages/sso-menu/src/SSOMenu.tsx

```diff
@@ -0,0 +1,250 @@
+  text-decoration: none;
+`;
+
+const menuItemPadding = css`
+  padding-left: 20px;
+`;
+
+const logoutContainerHeight = css`
+  height: 46px;
+`;
+
+const menuItems = [
+  {
+    displayName: 'Atlas',
+    description: 'cloud.mongodb.com',
+    href: 'https://cloud.mongodb.com',
+    slug: 'atlas',
+  },
+  {
+    displayName: 'University',
+    description: 'university.mongodb.com',
+    href: 'https://university.mongodb.com',
+    slug: 'university',
+  },
+  {
+    displayName: 'Cloud Support',
+    description: 'support.mongodb.com',
+    href: 'https://support.mongodb.com',
+    slug: 'support',
+  },
+] as const;
+
+export const ActiveProduct = {
+  Atlas: 'atlas',
+  University: 'university',
+  Support: 'support',
+} as const;
+
+type ActiveProduct = typeof ActiveProduct[keyof typeof ActiveProduct];
+
+interface SSOMenuProps {
+  /**
+   * Object that contains information about the active user. {name: 'string', email: 'string'}
+   */
+  user: { name: string; email: string };
+
+  /**
+   * MongoDB product that is currently active: ['atlas', 'university', 'support'].
+   */
+  activeProduct: ActiveProduct;
+
+  /**
+   * Callback invoked when user logs out.
+   */
+  onLogout?: React.MouseEventHandler;
+
+  /**
+   * Callback invoked when user switches products.
+   */
+  onProductChange?: React.MouseEventHandler;
+
+  /**
+   * Callback invoked when user views their MongoDB account.
+   */
+  onAccountClick?: React.MouseEventHandler;
+}
+
+/**
+ * # SSOMenu
+ *
+ * ```
+<SSOMenu
+    user={{ name: 'Alex Smith', email: 'alex.smith@youwork.com' }}
+    activeProduct="atlas"
+    onAccountClick={() => console.log('Viewing account information')}
+    onLogout={() => console.log('On logout')}
+    onProductChange={() => console.log('Switching products')}
+  />
+ * ```
+ * @param props.user Object that contains information about the active user. {name: 'string', email: 'string'}
+ * @param props.activeProduct  MongoDB product that is currently active: ['atlas', 'university', 'support'].
+ * @param props.onLogout Callback invoked when user logs out.
+ * @param props.onAccountClick Callback invoked when user views their MongoDB account.
+ * @param props.onProductChange Callback invoked when user switches products.
+ *
+ */
+function SSOMenu({
+  user: { name, email },
+  activeProduct,
+  onLogout,
+  onProductChange,
+  onAccountClick,
+}: SSOMenuProps) {
+  const triggerRef = useRef<HTMLButtonElement>(null);
+
+  return (
+    <Menu
+      trigger={
+        <button ref={triggerRef} type="button" className={menuButtonStyle}>
+          <span className={menuNameStyle}>{name}</span>
+          <Icon glyph="CaretDown" className={iconStyle} />
+        </button>
+      }
+    >
+      <MenuGroup className={accountMenuGroupStyle}>
+        <h3 className={cx(nameStyle, truncate)}>{name}</h3>
+        <p className={descriptionStyle}>{email}</p>
+        <Button
+          size="small"
+          onClick={onAccountClick}
```

Comment:
I think this will always redirect to account.mongodb.com, so we should add that `href` as well. Not sure we need the onClick handler.

Reply:
👍